{"name": "test_user_login_wrong", "status": "passed", "attachments": [{"name": "stdout", "source": "865bf6c2-98b9-4a5c-ab82-c741d55b2f42-attachment.txt", "type": "text/plain"}], "start": 1600238166981, "stop": 1600238167185, "uuid": "d6d1c076-8fdd-4354-912b-dff1b9b3455d", "historyId": "8923e36b93bfcbf2b1b29bba9c76fef6", "testCaseId": "4b6bcd6bc9f4b93fe085c2939884ea45", "fullName": "testcase.test_login.TestLogin#test_user_login_wrong", "labels": [{"name": "story", "value": "测试登录密码错误"}, {"name": "severity", "value": "critical"}, {"name": "feature", "value": "测试登录模块"}, {"name": "parentSuite", "value": "testcase"}, {"name": "suite", "value": "test_login"}, {"name": "subSuite", "value": "TestLogin"}, {"name": "host", "value": "TX5C3S1L7D31UYD"}, {"name": "thread", "value": "18760-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "testcase.test_login"}], "links": [{"type": "test_case", "url": "/basic/auth/login", "name": "/basic/auth/login"}]}